sensor_msgs/Image image
Rectangle[] rectangles